### ISSUE TITLE
Be (Way) More Invasive? 

### ISSUE THREAD

**Issue by trinitytakei** [fractaledmind/enlitenment#8]
I continued with the enlitement of my JSP-based app after #5 was fixed.

I ran into the next issue right away: the 'sqlite3' gem was not added to the Gemfile. Nor was the 'pg' gem removed. 

If the script should enlite apps that previously used another database gem (like pg in my case), then I'd consider this a bug and suggest that

1) the old db gem should be removed from Gemfile
2) sqlite3 should be added to Gemfile
3) (?) I just realized that for this to be fully automatic, the existing data should be imported from the original whatever-DB to the newly created SQLite DB... and that might be outside of the scope of this script? 

It's not white/black imo -  even if we consider 3) to be indeed out of scope, the script could still run 1) and 2), then display a message to the user to do this themselves ¯\\\_(ツ)\_/¯
